src/main/java/com/dougnoel/sentinel/elements/tables/Table.java

```diff
@@ -810,4 +810,16 @@ private String getCellsValue(List<Integer> indexes, List<String> cells) {
 		return cellValues;
 	}
 
+	/**
+	 * Verifies that the first column header is displayed before (to the left of) the second column header.
+	 * @param column1 String the name of the first column
+	 * @param column2 String the name of the second column
+	 * @return boolean true if the first column header's X coordinate is less than the second column header's X coordinate
+	 */
+	public boolean verifyColumnDisplayOrder(String column1, String column2){
```

Comment:
Curious why we're doing this in the table object itself? Is this really used so often?

Reply:
It doesn't have much to do with how often we're doing it. It lives in Table because the table knows this information about itself. It's in line with the existing design of Table functionality.

Reply:
Roger. Answers my question.

Reply:
For clarity I'll contrast this with the other step in this PR for element location: the reason the location of two elements should be checked in the step is because an element should not have to know information about any other element. To create a method in the Element class that compares one instance of Element to another instance of Element would be an anti-pattern. On the other hand, the columns are a part of the Table, and the Table should know any information related to its columns (and rows, cells, etc.). It makes sense that we can query the Table about the relative position of its columns and it can tell us "yes" or "no". Not the case for any given element in relation to some other element. So that's why I put the code where it is in each instance. Hope that helps clear things up!